### PR TITLE
Added a Android WebView sample demo

### PR DESCRIPTION
**Pull request by LeviSklaroff**
**Issue #:**
Implement a demo showcasing the Chime SDK inside of Android WebView

**Description of changes:**
Added the Android WebView sample demo that showcases how to run a WebRTC meeting demo inside of Android WebView

**Testing**
1. How did you test these changes?
- Clone the AndroidWebView branch from https://github.com/aws/amazon-chime-sdk-js
- Build the sdk
- Build the browser demo
- Deploy the serverless demo and copy the given url for later
- Launch the newly added Android-Webview-sample folder in Android Studio 4.2.2
- In the AppConfig.kt file replace PLACEHOLDER with the url from the serverless demo
- Connect a physical android device that is in developer mode
- Launch the app on the physical device
- On the android device press the 'Go to WebView' button
- Fill in the fields on the page that appears
- Click continue
- Click continue on the next page
- Open the serverless url on another device and browser, then perform the last 3 steps
- Test for audio input on the android device
- Test for audio output on the android device
- Test for video input on the android device

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
Yes, the serverless demo

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/Android-WebView-sample/Config [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        1.0 = {
+        };
+    };
+
+    runtime-dependencies = {
+        1.0 = {
+        };
+    };
+
+};
```
[the remark this answers is not in the document]
Comment (LeviSklaroff) [reply apps/Android-WebView-sample/Config]: Removed

File: apps/Android-WebView-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Amazon Chime SDK WKWebView Sample
+
+## Summary
+
+This sample shows how to run a WebRTC based meeting application inside of an Android Chromium WebView. The sample itself is a bare-bones Android app that loads a Chromium WebView, which then navigates to a specified URL. Currently Android Chromium WebView does not support grabbing media device label, so you may need to make some minor changes to your application in order for Chromium WebView to work correctly.
```
[the remark this answers is not in the document]
Comment (LeviSklaroff) [reply apps/Android-WebView-sample/README.md]: Changed

File: apps/Android-WebView-sample/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+### Pre-requisites:
+- Android Studio 4.2.2+ installed on your machine
+
+### Deploy a Chime SDK for Javascript Meeting Demo
+If you'd like to try joining a WebRTC meeting demo from within a Chromium WebView in Android, you first need a meeting demo running in a browser so that when you load the WKWebView it can navigate to this meeting demo URL. Follow the instructions for the [Amazon Chime JS SDK Serverless Meeting Demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/serverless) in order to deploy a meeting demo.
+
+After you've deployed a meeting demo URL, take note of the meeting demo URL that was outputted to your terminal as part of the last step of deploying the [Amazon Chime SDK for Javascript Serverless Meeting Demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/serverless). Take the meeting demo URL, and replace the variable `url` in `./AppConfig.java` with the value of the meeting demo URL.
```
[the remark this answers is not in the document]
Comment (LeviSklaroff) [reply apps/Android-WebView-sample/README.md]: Changed

File: apps/Android-WebView-sample/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  -->
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.amazonaws.android_webview_sample">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
+
```
[the remark this answers is not in the document]
Comment (LeviSklaroff) [reply apps/Android-WebView-sample/app/src/main/AndroidManifest.xml:17]: Added a comment above

File: apps/Android-WebView-sample/app/src/main/java/com/amazonaws/android_webview_sample/AppConfig.kt [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+ * AppConfig.kt
+ * Android WebView Sample Demo
+ *
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT-0
+ */
+package com.amazonaws.android_webview_sample
+
+object AppConfig {
+    // PLEASE PUT YOUR MEETING URL HERE
```
[the remark this answers is not in the document]
Comment (LeviSklaroff) [reply apps/Android-WebView-sample/app/src/main/java/com/amazonaws/android_webview_sample/AppConfig.kt]: Changed to simply PUT YOUR MEETING URL HERE

**Review by michhyun1 — APPROVED** [review]

**Pull request merged**